bt.sh

```diff
@@ -228,7 +232,7 @@ bt_report () {
   fi
 
   # measurements sorted chronologically by start time
-  for m in $(ls -1 /tmp/bt.*.* | sort -t '.' -k3,3 -n); do
+  for m in $(ls -1 "$BT_DIR"/*.* | sort -t '.' -k3,3 -n); do
```

Comment:
Hey y'all, long time no see 😊. Thanks for this PR @piki: I've incorporated it into https://github.com/urcomputeringpal/checkrun-timechart-action. This one change was required to retain the previous existing sorting behavior:

```suggestion
  for m in $(ls -1 "$BT_DIR"/*.* | sort -t '.' -k2,2 -n); do
```

Reply:
Good catch.  👍